src/requests/features.jl

```diff
@@ -300,14 +300,23 @@ function _binding_kind(b, server)
     end
 end
 
-function julia_getModuleAt_request(params::TextDocumentPositionParams, server::LanguageServerInstance, conn)
-    doc = getdocument(server, URI2(params.textDocument.uri))
-    offset = get_offset(doc, params.position)
-    x = get_expr(getcst(doc), offset)
-    if x isa EXPR
-        scope = StaticLint.retrieve_scope(x)
-        if scope !== nothing
-            return get_module_of(scope)
+function julia_getModuleAt_request(params::VersionedTextDocumentPositionParams, server::LanguageServerInstance, conn)
+    uri = URI2(params.textDocument.uri)
+
+    if hasdocument(server, uri)
```

Comment:
I don't think we should have this check here. If seems to me that we have an error if the doc doesn't exist in the LS, so we should just crash in that situation to make sure we learn of the problem and can then fix it. I would just remove this `if` clause here, so that `getdocument` crashes if the doc doesn't exist.

Reply:
Hm, I don't think that's a good idea -- we can't ever guarantee in which order our custom requests are interleaved with the LS internals, right?

Reply:
Yeah, I guess that is true... We should probably return a `JSONRPCError` then and handle that situation in the client?